File: 1._Belajar_JavaScript_Console/1._Latihan_Variasi/Latihan_Input_Menentukan_Persegi_dan_Persegi_Panjang.js
// Program menentukan bentuk bangun datar (2D)

console.log ("\n Program menentukan bentuk bangun datar (2D) \n")


var a = Number (prompt ("Sisi 1 : "))
var b = Number (prompt ("Sisi 2 : "))
var c = Number (prompt ("Sisi 3: "))
var d = Number (prompt ("Sisi 4 : "))


if ((a > 0) && (b > 0) && (c > 0) && (d > 0)) {
    
    if (a == b == c == d) {
        console.log ("Persegi")
    }
    
    else if ((a == c && b == d) || (a == b && c == d) || (a == d && a == c)) {
        console.log ("Persegi panjang")
    }
    
    else {
        console.log ("Bentuk lain")
    }
}

else {
    console.log ("Gak boleh negatif dan 0")
}

File: 1._Belajar_Python/1._Latihan_Variasi/Latihan_Input_Menentukan_Persegi_dan_Persegi_Panjang.py
# Program menentukan bentuk bangun datar (2D)

print ("\n Program menentukan bentuk bangun datar (2D) \n")


a = int (input ("Sisi 1 :"))
b = int (input ("Sisi 2 :"))
c = int (input ("Sisi 3 :"))
d = int (input ("Sisi 4 :"))


if a > 0 and b > 0 and c > 0 and d > 0:
    
    if a == b == c == d:
        print ("Persegi")
        
    elif ((a == c and b == d) or (a == b and c == d) or (a == d and a == c)):
        print ("Persegi Panjang")
        
    else:
        print ("Bentuk lain")
        
else:
    print ("Gak boleh negatif atau 0")